src/utils/createContextWrapper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+/**
+ * Creates new trigger class that injects context into overlay.
+ */
+export default function createContextWrapper(Trigger, propName) {
+  return function (contextTypes) {
+    class ContextWrapper extends React.Component {
```

Comment:
Just because we aren't documenting it right now does not mean that we won't later. Once React starts documenting it, then we should too.

I know I said no documentation, but maybe we should add some comments to the code explaining how to use this. So, only those that really dig in will know how it's intended to be used.